openreview/conference/builder.py

```diff
@@ -171,6 +171,8 @@ def __expire_invitations(self, name):
         return len(invitations)
 
     def __create_submission_stage(self):
+        if self.submission_stage.due_date and self.submission_stage.due_date < datetime.datetime.utcnow():
```

Comment:
Could you avoid the if statement calling `set_submission_invitation(self, not(self.submission_stage.due_date and self.submission_stage.due_date < datetime.datetime.utcnow()))`

or 

```
under_submission=not self.submission_stage.due_date or self.submission_stage.due_date > datetime.datetime.utcnow()
set_submission_invitation(self, under_submission=under_submission)
```